multi-node/generic/controller-install.sh

```diff
@@ -5,7 +5,10 @@ set -e
 export ETCD_ENDPOINTS=
 
 # Specify the version (vX.Y.Z) of Kubernetes assets to deploy
-export K8S_VER=v1.2.2_coreos.0
+export K8S_VER=v1.2.3_coreos.0-cni
+
+# Hyperkube image repository to use.
+export HYPERKUBE_IMAGE_REPO=quay.io/calico/hyperkube
```

Comment:
I'm sticking with the _REPO suffix (unless you feel strongly about it) as that's what's already used for AWS.